File: javascript/leetcode/389. Find the Difference.js
/**
 * @param {string} s
 * @param {string} t
 * @return {character}
 */
var findTheDifference = function(s, t) {
  let sum_s = 0;
  let sum_t = 0;

  for (let i = 0; i < s.length; i++) {
    sum_s += s.charCodeAt(i);
  }

  for (let i = 0; i < t.length; i++) {
    sum_t += t.charCodeAt(i);
  }

  return String.fromCharCode(sum_t - sum_s);
};

File: go/leetcode/389.Find the Difference.go
func findTheDifference(s string, t string) byte {
    sum_s := byte(0)
    sum_t := byte(0)
    
    for i := 0; i < len(s); i++ {
        sum_s += (s[i])
    }
    
    for i := 0; i < len(t); i++ {
        sum_t += t[i]
    }
    
    return sum_t - sum_s
}
